### PR TITLE
Fix Color Normalized Issue

### PR DESCRIPTION
**Pull request by mingloo**
I've found the `self.pixel_means` is changed in every iteration when calling `__getitem__` due to modify of `mean` variable in `color_normalize` function

See detailed log below:
> checking pixel means __getitem__: [122.7717 115.9465 102.9801]
> checking pixel means __getitem__: [0.48145765 0.45469216 0.40384353]
> checking pixel means __getitem__: [0.00188807 0.00178311 0.0015837 ]
> checking pixel means __getitem__: [7.40419296e-06 6.99257450e-06 6.21058869e-06]
> checking pixel means __getitem__: [2.90360508e-08 2.74218608e-08 2.43552498e-08]

### CODE REVIEW

**Comment by GengDavid** [comment]
@mingloo Thanks for contribution. I'm sorry to have this bug. This will result in training and testing without normalization after a few iterations. However, if I merge your code now, all pre-trained models will fail to use. I'm going to fine-tune all models, and I'll merge your PR as soon as I finish it. 
Thank again.

**Comment by mingloo** [comment]
@GengDavid No problem. Looking forward to your update.

**Comment by mingloo** [comment]
Hi @GengDavid if you start to retrain all these model, will you includes this changes https://github.com/GengDavid/pytorch-cpn/issues/7 as well?

**Comment by GengDavid** [comment]
Right. I've started fine-tune work with issue #7 included.

**Comment by mingloo** [comment]
@GengDavid Cool.

**Comment by mingloo** [comment]
@GengDavid Thanks 👍
